security/advancedtls/advancedtls.go

```diff
@@ -261,9 +213,11 @@ type ServerOptions struct {
 	// RootOptions is OPTIONAL on server side. This field only needs to be set if
 	// mutual authentication is required(RequireClientCert is true).
 	RootOptions RootCertificateOptions
-	// If the server want the client to send certificates.
+	// If the server want the client to send certificates. This value is only
```

Comment:
want -> wants.  Or "requires" probably is more precise.

Reply:
Fixed